### DISCUSSION TITLE
Is it possible to rebase open PRs without running the usal repo run

### DISCUSSION THREAD

**Discussion by NicklasWallgren** [renovatebot/renovate#28317] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Is there an option to rebase open PRs without running the usal repo run?

We would like to reduce the noise and only schedule new PRs once a week - but still allow rebasing of existing ones every x minute.

**Comment by rarkins** [comment]
By default Renovate will reuse existing PRs

**Reply by NicklasWallgren** [reply]
Sorry - I was unclear in my question. We would like to run Renovate in a rebase-only mode for already existing PR:s, similar to the question below - and skip creating new PR:s for other dependencies that needs updating.

Our end game is to rebase open PR:s every x minute - and create new ones once a week.
https://github.com/renovatebot/config-help/issues/722#issuecomment-650134924

Could you please eloborate a bit more regarding your answer here - how would one force override a schedule?
https://github.com/renovatebot/config-help/issues/722#issuecomment-650213580

**Reply by rarkins** [reply]
I still don't understand what you're asking and that question is 4 years old

**Reply by NicklasWallgren** [reply]
In GitLab you can request Renovate to rebase using the checkbox - the issue is that the rebase won't trigger until the next time Renovate runs. 

How would one schedule Renovate to rebase those PR:s in shorter intervals / or directly without executing the "usual" repo run.

With Dependabot we can simple run `@dependabot rebase` and it will be triggered directly, but with our current setup of Renovate we need to wait until the next time Renovate starts - which is currently configured to run once a week.

Is there support for webhooks, or can we schedule renovate to just rebase those PR:s in shorter intervals (without running the "usual" repo run)?

**Reply by rarkins** [reply]
Something that runs in your pipeline can never respond to webhooks. Instead you should try https://www.mend.io/renovate-community/ https://github.com/mend/renovate-ce-ee

**Comment by codler** [comment, marked as answer]
You can add a Renovate Gitlab Pipeline Job for only MR with rebase-only settings.

**Reply by NicklasWallgren** [reply]
Thanks - we have resolved the issue 🙂

**Reply by Dahlorna** [reply]
@codler can you give an example of rebase-only settings?
